docs/howto/use-with-samba.md

```diff
@@ -0,0 +1,239 @@
+   server's hostname or IP address):
+
+   ```bash
+   smbclient //$SERVER/alice -U alice
+   ```
+
+   Enter the Samba password for `alice` when prompted. If successful, a `smb: \>`
+   prompt appears.
+
+3. **Mount the Share:**
+   Create a mount point for the share:
+
+   ```bash
+   mkdir -p /home/alice/samba
+   ```
+
+   Mount the share using the `cifs` filesystem type:
+
+   ```bash
+   sudo mount -t cifs //$SERVER/alice /home/alice/samba -o user=alice,uid=$(id -u alice),gid=$(id -g alice)
+   ```
+
+   Enter the Samba password for `alice` when prompted.
+
+4. **Optional: Add to `/etc/fstab` for Persistent Mounting:**
+   To automatically mount the share at boot, use a credentials file:
+
+   - Create a credentials file:
+
+     ```bash
+     sudo editor /etc/samba/credentials
+     ```
+
+     Add the following lines:
+
+     ```
+     username=alice
+     password=YOUR_PASSWORD
+     ```
+
+   - Secure the credentials file:
+
+     ```bash
+     sudo chmod 600 /etc/samba/credentials
+     ```
+
+   - Update `/etc/fstab`:
+
+     ```
+     //$SERVER/alice /home/alice/samba cifs credentials=/etc/samba/credentials,uid=alice,gid=alice 0 0
+     ```
+
+5. **Verify the Mount:**
+   As the user `alice`, try accessing the shared directory:
+
+   ```bash
+   ls -la /home/alice/samba
+   ```
+
+   Verify write access by creating a test file:
+
+   ```bash
+   touch /home/alice/samba/test
+   ```
+
+   **Security Note:** Files and directories in the share may appear as owned by
+   `alice` on the client, but access control is enforced by the server. For
+   example:
+
+   - If `alice` does not have permission on the server, access will be denied
+     even if ownership appears correct on the client.
+
+   Example:
+
+   - Create a restricted directory on the server:
+
+     ```bash
+     sudo mkdir /srv/samba/alice/secrets
+     sudo chmod 700 /srv/samba/alice/secrets
+     ```
+
+   - Attempt to access it on the client:
+
+     ```bash
+     ls /home/alice/samba/secrets
+     ```
+
+     **Result:** *Permission denied.*
+
+---
+
+### Cleanup
+
+#### On the Server
+
+1. **Delete the Shared Directory:**
+   Remove the directory used for the Samba share:
+
+   ```bash
+   sudo rm -rf /srv/samba/alice
+   ```
+
+2. **Purge Installed Samba Packages:**
+   If Samba is no longer needed, uninstall it completely:
+
+   ```bash
+   sudo apt purge samba samba-common
+   sudo apt autoremove
+   ```
+
+---
+
+#### On the Client
+
+1. **Unmount the Shared Directory:**
+   Unmount the shared directory:
+
+   ```bash
+   sudo umount /home/alice/samba
+   ```
+
+2. **Delete the Mount Point:**
+   Remove the mount point directory:
+
+   ```bash
+   rmdir /home/alice/samba
+   ```
+
+3. **Remove Entry from `/etc/fstab`:**
+   If you added the share to `/etc/fstab`, remove its entry:
+
+   ```bash
+   sudo editor /etc/fstab
+   ```
+
+   Locate and delete the line referencing the Samba share, then save and close.
```

Comment:
If the user has followed this guide closely, we should be able to specify exactly the line that they need to locate and delete?

Reply:
How would you specify that? Like this?

 Locate and delete the Samba share entry:

   ```
   //$SERVER/alice /home/alice/samba cifs credentials=/etc/samba/credentials,uid=alice,gid=alice 0 0
   ```

Reply:
Let's not add it: it's probably obvious to someone following the guide.